verification/cbmc/proofs/aws_mul_size_checked/aws_mul_size_checked_harness.c

```diff
@@ -24,7 +24,7 @@ void aws_mul_size_checked_harness() {
         if (!aws_mul_u64_checked(a, b, &r)) {
             assert(r == a * b);
         } else {
-            assert(__CPROVER_overflow_mult(a, b));
```

Comment:
I think this is backwards. You want to use this as the check, but you want to make sure that you're linking against the actual C check in the math.h

Reply:
Yep, I can disable the CBMC flag in this proof.